bcgov/biohubbc
# BHBC-1837: Fix survey attachment metadata endpoint

# Overview

## Links to Jira tickets

https://quartech.atlassian.net/browse/BHBC-1837

## Description of relevant changes

Fix survey attachment metadata metadata endpoint.

Also included some very minor tweaks to the openapi spec for the metadata endpoints (while I was there already), and added an extra debug log to the response validation function so we can actually see the response body that was invalid, not just the validation messages.

To test:
- Create project/survey
- Upload a Report
- Click the 3 dot menu -> View details
- See that the popup opens successfully (no request error)

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code


### Style


### Documentation


### Tests


### Linting/Formatting

       _See the `lint` commands in package.json_
       _See the `format` commands in package.json_

### SonarCloud
